src/nsDefinitions.h

```diff
@@ -21,4 +21,4 @@ under the License.
 #define COMMON_NS common
 #define RESQML2_NS resqml2
 #define RESQML2_0_1_NS resqml2_0_1
-#define WITSML1_4_1_1_NS witsml1_4_1_1
+#define WITSML2_0_NS witsml2_0
```

Comment:
I think this file should be deleted from git. It should be generated by cmake.

Reply:
Fixed.